backend/app/api/v1/config_definition.py

```
@@ -135,19 +141,30 @@ def list_config_definition(page: int = 1, limit: int = 10):
     ListConfigDefinitionResponse
         The response for the list configuration definition request.
     """
     try:
-        config_definitions = l_config_definition(page, limit)
+        config_definitions, count = l_config_definition(
+            page, limit, sort_by, sort_order, search
+        )
 
     except Exception as e:
         logger.exception(f"Error listing configuration definitions: {e}")
         raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)